benchmarking/mod/bblock/vdbench.py

```diff
@@ -97,13 +97,54 @@ def prepare_run(self):
         self.cleanup()
 	# format
 	self.format_run()
-
+	#Add new method to check vdbench
+	def check_vdbench_pgrep(self, nodes, vdbench_node_num = 1, check_type="jobnum"):
+        user =  self.cluster["user"]
+        stdout, stderr = common.pdsh(user, nodes, "pgrep -x vdbench", option = "check_return")
```

Comment:
here use "pgrep -x java" would be better. After killing vdbench process, java test is still running. But after killing java process, vdbench is finished too.